pyvista/core/filters/uniform_grid.py

```diff
@@ -414,3 +414,265 @@ def image_threshold(
+
+        """
+        # check for active scalars, otherwise risk of segfault
+        if self.point_data.active_scalars_name is None:
+            raise ValueError('FFT filter requires active point scalars')
+
+        alg = _vtk.vtkImageFFT()
+        alg.SetInputDataObject(self)
+        _update_alg(alg, progress_bar, 'Performing Fast Fourier Transform')
+        output = _get_output(alg)
+        self._change_output_scalars(output, output_scalars_name)
+        return output
+
+    def rfft(self, output_scalars_name='ImageScalars', progress_bar=False):
+        """Apply the reverse fast fourier transform to the active scalars.
+
+        The input can have real or complex data in any components and data
+        types, but the output is always complex doubles, with the first
+        component containing real values and the second component containing
+        imaginary values. The filter is fastest for images that have power of
+        two sizes.
+
+        The filter uses a butterfly diagram for each prime factor of the
+        dimension. This makes images with prime number dimensions (i.e. 17x17)
+        much slower to compute. Multi dimensional (i.e volumes) FFT's are
+        decomposed so that each axis executes serially.
+
+        Parameters
+        ----------
+        output_scalars_name : str, optional
+            The name of the output scalars. By default this is
+            ``'ImageScalars'``.
+
+        progress_bar : bool, optional
+            Display a progress bar to indicate progress.
+
+        Returns
+        -------
+        pyvista.UniformGrid
+            UniformGrid with the applied reverse FFT.
+
+        Examples
+        --------
+        Apply reverse FFT to an example image.
+
+        >>> from pyvista import examples
+        >>> image = examples.download_moonlanding_image()
+        >>> fft_image = image.fft()
+        >>> image_again = fft_image.rfft()
+        >>> image_again.point_data  # doctest:+SKIP
+        pyvista DataSetAttributes
+        Association     : POINT
+        Active Scalars  : PNGImage
+        Active Vectors  : None
+        Active Texture  : None
+        Active Normals  : None
+        Contains arrays :
+        PNGImage                float64  (298620, 2)          SCALARS
+
+        See :ref:`image_fft_example` for a full example using this filter.
+
+        """
+        self._check_fft_scalars()
+        alg = _vtk.vtkImageRFFT()
+        alg.SetInputDataObject(self)
+        _update_alg(alg, progress_bar, 'Performing Reverse Fast Fourier Transform.')
+        output = _get_output(alg)
+        self._change_output_scalars(output, output_scalars_name)
+        return output
+
+    def low_pass(
+        self,
+        x_cutoff,
+        y_cutoff,
+        z_cutoff,
+        order=1,
+        output_scalars_name='ImageScalars',
+        progress_bar=False,
+    ):
+        """Perform a low pass filter in the frequency domain.
```

Comment:
Should we mention somewhere that this uses a Butterworth filter? The `order` parameter makes me lean toward "yes".

Reply:
Fixed in 2b26cccfc95ac76c5c082c82258e360aa64712d7